docs/content/docs/tasks_usage/namespace_monitoring.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Namespace Monitoring"
+description: "This section describes how and why OSM monitors Kubernetes namespaces"
+type: docs
+---
+
+## Overview
+
+When setting up an OSM control plane (also referred to as a "mesh"), one can also enroll a set of Kubernetes namespaces to the mesh. Enrolling a namespace to OSM allows OSM to monitor the resources within that Namespace whether they be applications deployed in Pods, Services, or even traffic policies represented as SMI resources.
+
+Only one mesh can monitor a namespace, so this is something to watch out for when there are multiple instances of OSM within the same Kubernetes cluster. When applying policies to applications, OSM will only assess resources in either monitored namespaces so it is important to enroll namespaces where your applications are deployed to the correct instance of OSM with the correct mesh name.
+Enrolling a namespace also optionally allows for metrics to be collected for resources in the given namespace and for Pods in the namespace to be automatically injected with sidecar proxy containers. These are all features that help OSM provide functionality for traffic management and observability. Scoping this functionality at the namespace level allows teams to organize which segments of
+their cluster should be part of which mesh.
+
+Namespace monitoring, automatic sidecar injection, and metrics collection is controlled by adding certain labels and annotations to a Kubernetes namespace. This can be done manually or using the `osm` CLI although using the `osm` CLI is the recommended approach. The presence of the label `openservicemesh.io/monitored-by=<mesh-name>` allows an OSM control place with the given `mesh-name` to monitor
+all resources within that namespace. The annotation `openservicemesh.io/sidecar-injection=enabled` enables OSM to automatically inject sidecar proxy containers in all Pods created within that namespace. The metrics annotation `openservicemesh.io/metrics=enabled` allows OSM to collect metrics on resources within a Namespace.
+
+See how to use the OSM CLI to manage namespace monitoring below.
+
+## Adding a Namespace to the OSM Control Plane
+
+Add a namespace for monitoring and sidecar injection to the mesh with the following command:
+
+```bash
+osm namespace add <mesh-name>
+```
+
+Explicitly disable sidecar injection while adding the namespace using `--disable-sidecar-injection flag as show [here](../sidecar_injection.md#Explicitly-Disabling-Automatic-Sidecar-Injection-on-Namespaces).
```

Comment:
The link seems to be giving a 404 for me. Not sure if it's going one dir too far back?
```suggestion
Explicitly disable sidecar injection while adding the namespace using `--disable-sidecar-injection flag as show [here](./sidecar_injection.md#Explicitly-Disabling-Automatic-Sidecar-Injection-on-Namespaces).
```

Reply:
Thank you!